framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -505,6 +518,29 @@ private void parseMethod(MethodDeclaration decl, ExecutableElement elt,
         putNew(atypes, elt, methodType);
     }
 
+    private void handleExistingAnnotations(AnnotatedTypeMirror atype, Type typeDef) {
```

Comment:
Will do.
Can you suggest a good location in the Checker Framework manual where this changed behavior can be documented?